api/app/applicationModel/api.py

```diff
@@ -130,21 +130,19 @@ def post(self):
                 is_open,
                 nominations
             )
-            db.session.add()
+            db.session.add(app_form)
 
         section_args = args['section']
-        section = Section(section_args)
-
-        question_args = args['question']
-        question = Question(question_args)
-
-        for s in section: # TODO: Confirm what is being looped over: is this all the new sections coming in from the front end?
+        for s in section_args: # TODO: Confirm this makes sense / works
+            section = Section(s)
             section = section(app_form.id)
-            db.session.add()
+            db.session.add(section)
 
-        for q in question: # TODO: reassess logic for loop (same question as above)
-            question = q(app_form.id, section.id)
-            db.session.add()
+        question_args = args['question']
+        for q in question_args: # TODO: Confirm this makes sense / works
+            question = Question(q)
```

Comment:
This isn't how you construct an object in Python?

Reply:
Thank you for checking! I can't actually explain what I was trying to achieve - I _think_ I have a better idea today though!